adjust readme for v2 release (#612)
SalesforceCommerceCloud/pwa-kit: packages/pwa-kit-dev/README.md

> # PWA Kit Developer Tools 🛠️

Reviewer: ```suggestion
# PWA Kit Developer Tools
```

For consistency with other README files in our repo, use emoji only on h2s (and put the emoji before the heading text, not after).

> A set of commands to build PWA Kit projects

Reviewer 2: ```suggestion
A command line interface tool to develop, build and deploy PWA Kit projects.
```

> A command line interface tool to develop, build and deploy PWA Kit projects.

Reviewer: ```suggestion
A command line interface tool to develop, build, and deploy PWA Kit projects.
```

## Requirements

```
  Node ^14.x
  npm ^6.14.4
```

> For see all the available commands, run:

Reviewer 2: ```suggestion
To see all the available commands:
```
